frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -909,12 +909,12 @@ export class HtmlDocument extends React.Component {
 
               {state.mode === APP_FEATURE_MODE.REVISION && (
                 <button
-                  className='wsContentGeneric__option__menu__lastversion html-document__lastversionbtn btn highlightBtn'
+                  className='wsContentGeneric__option__menu__lastversion html-document__lastversionbtn btn'
                   onClick={this.handleClickLastVersion}
                   style={{ backgroundColor: state.config.hexcolor, color: '#fdfdfd' }}
                 >
                   <i className='fas fa-history' />
-                  {props.t('Last version')}
+                  <span>{props.t('Last version')}</span>
                 </button>
               )}
```

Comment:
We may want to open a technical dept issue  to remove the duplication in apps (duplicated here: https://github.com/tracim/tracim/pull/4670/files#diff-391722c17977af4e654df7a5c6f6b36798e660c0f821ab2910cbc7f8b70c3240R1141)

Reply:
I agree, I think this specific repetition only happens twice though.